airbyte-integrations/connectors/source-freshdesk/README.md

```diff
@@ -0,0 +1,64 @@
+# Freshdesk Source 
+
+This is the repository for the Freshdesk source connector, written in Python. 
+For information about how to use this connector within Airbyte, see [the documentation](https://docs.airbyte.io/integrations/sources/freshdesk).
+
+## Local development
+
+### Prerequisites
+**To iterate on this connector, make sure to complete this prerequisites section.**
+
+#### Build & Activate Virtual Environment
+First, build the module by running the following from the `airbyte` project root directory: 
+```
+./gradlew :airbyte-integrations:connectors:source-freshdesk:build
+```
+
+This will generate a virtualenv for this module in `source-freshdesk/.venv`. Make sure this venv is active in your
+development environment of choice. To activate the venv from the terminal, run:
+```
+cd airbyte-integrations/connectors/source-freshdesk # cd into the connector directory
+source .venv/bin/activate
+```
+If you are in an IDE, follow your IDE's instructions to activate the virtualenv.
+
+#### Create credentials
+**If you are a community contributor**, follow the instructions in the [documentation](https://docs.airbyte.io/integrations/sources/{{dashCase name}})
+to generate the necessary credentials. Then create a file `secrets/config.json` conforming to the `source_{{snakeCase name}}/spec.json` file.
+See `sample_files/sample_config.json` for a sample config file.
+
+**If you are an Airbyte core member**, copy the credentials in RPass under the secret name `source-{{dashCase name}}-integration-test-config`
+and place them into `secrets/config.json`.
+
+
+### Locally running the connector
+```
+python main_dev.py spec
+python main_dev.py check --config secrets/config.json
+python main_dev.py discover --config secrets/config.json
+python main_dev.py read --config secrets/config.json --catalog sample_files/configured_catalog.json
+```
+
+### Unit Tests
+To run unit tests locally, from the connector directory run:
+```
+pytest unit_tests
+```
+
+### Locally running the connector docker image
+```
+# in airbyte root directory
+./gradlew :airbyte-integrations:connectors:source-freshdesk:airbyteDocker
+docker run --rm airbyte/source-freshdesk:dev spec
+docker run --rm -v $(pwd)/airbyte-integrations/connectors/source-freshdesk/secrets:/secrets airbyte/source-freshdesk:dev check --config /secrets/config.json
+docker run --rm -v $(pwd)/airbyte-integrations/connectors/source-freshdesk/secrets:/secrets airbyte/source-freshdesk:dev discover --config /secrets/config.json
+docker run --rm -v $(pwd)/airbyte-integrations/connectors/source-freshdesk/secrets:/secrets -v $(pwd)/airbyte-integrations/connectors/source-freshdesk/sample_files:/sample_files airbyte/source-freshdesk:dev read --config /secrets/config.json --catalog /sample_files/configured_catalog.json
```

Comment:
`configured_catalog` doesn't exist -- could you do a pass on this readme to make sure the file names are correct?